generate-thumbnail/functions/index.js

```diff
@@ -26,20 +28,26 @@ const THUMB_MAX_HEIGHT = 200;
 const THUMB_MAX_WIDTH = 200;
 // Thumbnail prefix added to file names.
 const THUMB_PREFIX = 'thumb_';
+admin.initializeApp(functions.config().firebase);
+const ref = admin.database().ref();
 
 /**
  * When an image is uploaded in the Storage bucket We generate a thumbnail automatically using
  * ImageMagick.
```

Comment:
Could you add a sentence about the signedURL in the function comment here? something like: "After the thumbnail has been generated and uploaded to Cloud Storage we write the public URL to the Firebase Realtime Database."